### PR TITLE
Add advocate_origin variable

### PR DESCRIPTION
**Pull request by Tonkonozhenko**
image

### CODE REVIEW

File: source/campaigns/editor/variables.rst [hunk 1]
```diff
@@ -59,6 +59,33 @@ Personal data for |advocate|.
...
+
+   ==================== ================================= ==================================================================
+   Property             Value                             Description
+   ==================== ================================= ==================================================================
+   id                   1234567890                        Advocate origin id
+   new_customer         false | true                      Advocate is a new customer or not
+   created_at           "2016-07-27T00:00:00.000+03:00"   Time when Advocate origin was created
+   event_category       "purchase"                        Advocate origin event category
+   coupon_codes         ["AD_DISCOUNT"]                   Coupon codes that are used in Advocate origin
+   location             {                                 Advocate Location regarding his IP address
```
Comment (iurevych) [inline source/campaigns/editor/variables.rst]: "Advocate Location regarding his IP address" -> "Advocate geolocation based on their IP address"

**Comment by iurevych** [comment]
Just one copy suggestion and then looks good

**Comment by iurevych** [comment]
Hmm I think it also makes sense to clarify what is Origin. Maybe even make sense to add a hover over explanation to it to then re-use in other places.

**Comment by Tonkonozhenko** [comment]
It will look like this. It's not very nice, but it's the best solution to make tests pass.

image

**Comment by iurevych** [comment]
Looks bad, but what we can do if we can't fix it

**Pull request merged**